packages/components/src/DescriptionList/DescriptionList.test.tsx

```diff
@@ -14,6 +14,38 @@ it("renders an object as a list of key value pairs", () => {
   expect(container).toMatchSnapshot();
 });
 
+it("allows duplicate terms", () => {
+  const { getAllByText } = render(
+    <DescriptionList
+      data={[
+        ["Issued", "2018-12-08"],
+        ["Issued", "2019-01-06"],
+      ]}
+    />,
+  );
+  expect(getAllByText("Issued")).toHaveLength(2);
+});
+
+it("does not throw a key error when duplicate terms are used", () => {
+  const logSpy = jest.spyOn(global.console, "error");
+
+  render(
+    <DescriptionList
+      data={[
+        ["Issued", "2018-12-08"],
+        ["Issued", "2019-01-06"],
+      ]}
+    />,
+  );
+  expect(logSpy).not.toHaveBeenCalledWith(
+    expect.stringContaining(
```

Comment:
@darryltec This is what I came up with. This seems like a strange test to me, though. However, it does protect against a regression. Let me know if this is what you were thinking.

Reply:
Perfection!